Life-Pill/pharmacy-pos-frontend
# Fix Still Image Adding Issue and Enhance Image Handling in Cashier View

# Pull Request: Fix Still Image Adding Issue and Enhance Image Handling in Cashier View

## Description
Fixes an issue where adding still images was not functioning correctly while ensuring images display correctly in the cashier view.

## Changes Made
- Addressed issue causing still image adding to fail.
- Enhanced image handling to ensure images are fetched and displayed correctly in the cashier view.

### Commits Included:
- `aws bucket update is done` by @TheTharz
- `fetching the updated image` by @TheTharz
- `view cashier is done` by @TheTharz
- `fetching and displaying image is done` by @TheTharz
- `all the functions are done in cashier` by @TheTharz
- `view cashier problem with image fixed` by @TheTharz

## Manual Testing Done
1. Tested adding still images to verify functionality.
2. Checked that images are fetched and displayed correctly in the cashier view.
3. Verified that no regressions were introduced in other areas of the application.

## Additional Information
- Name: Tharindu Jayawardhana
- Position: Frontend Developer at LifePill
